src/common/components/collections/CollectionPreviewCard.tsx

```diff
@@ -0,0 +1,29 @@
+import Link from 'next/link';
+import React from 'react';
+
+interface CollectionCardProps {
+  collectionAddress: string;
```

Comment:
just a tiny comment: you shouldnt prefix everything with `collection` inside an interface called `CollectionCardProps`. Better for the default to be that it's part of the collection, and to prefix anything that isnt (e.g. `ownerAddress`)

Reply:
In general I agree, but because we do a lot of destructuring in various places it quite often becomes clearer when you have the prefix! Of course, we can just rename it in those instances. I'll take a note to refactor these soon :)